Core/Rewriters/Formatter.cs

```diff
@@ -143,11 +143,27 @@ private static InvocationExpressionSyntax FormatInvocationExpression (Invocation
 
     private static MemberAccessExpressionSyntax FormatMemberAccessExpression (MemberAccessExpressionSyntax node)
     {
-      var indentation = ((MemberAccessExpressionSyntax) node.DescendantNodes().First (s => s.IsKind (SyntaxKind.SimpleMemberAccessExpression))).OperatorToken.LeadingTrivia.ToString();
+      var indentation = GetIndentation (node);
       return MoqSyntaxFactory.MemberAccessExpression (
           node.Expression.WithTrailingTrivia (SyntaxFactory.Whitespace (Environment.NewLine + indentation)),
           node.Name);
     }
+    private static string GetIndentation (MemberAccessExpressionSyntax node)
+    {
+      var firstMemberAccessExpressionTrivia = ((MemberAccessExpressionSyntax) node.GetFirstIdentifierName().Parent!).OperatorToken.LeadingTrivia.ToFullString();
+      var secondMemberAccessExpressionTrivia = (node.GetFirstIdentifierName().Parent!.Parent!.Parent as MemberAccessExpressionSyntax)?.OperatorToken.LeadingTrivia.ToFullString();
```

Comment:
You can move the `secondMemberAccessExpressionTrivia` definition right between the 2 if statements following.